template.yml

```diff
@@ -88,7 +88,9 @@ jobs:
     file: 1-click/tasks/bosh-create-env.yml
     params:
       BOSH_LITE_NAME: ((bosh_lite_name))
-      MANIFEST: {{bosh-manifest}}
+      MANIFEST: ((bosh-manifest))
+      DOMAIN_NAME: ((domain_name))
```

Comment:
Does this break existing scripts?

Reply:
Yes. The idea was to always require the domain name instead of forcing the world to use "flintstone.ams" ;-) But I guess a default would also be possible.